src/nxdoc/nuxeo-server/installation/docker-image/build-a-custom-docker-image.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Build a Custom Docker Image
+review:
+  date: '2021-04-23'
+  status: ok
+labels:
+  - multiexcerpt
+  - multiexcerpt-include
+toc: true
+description: Discover how to build a custom Docker image.
+tree_item_index: 100
+---
+
+To build an application from Nuxeo, we strongly recommend to customize the Nuxeo Docker Image by building a Docker image from the Nuxeo one. Let's see how you can do this.
+
+## Build a Custom Docker Image From the Nuxeo One
+
+You can simply write a [Dockerfile](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) using the Nuxeo image as [parent image](https://docs.docker.com/glossary/#parent_image).
+A good practice is to use a [build argument](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact) before the `FROM` instruction to easily set the version of the Nuxeo parent image, as in the example below:
+
+```Dockerfile
+ARG NUXEO_VERSION=latest
```

Comment:
It should be 10.10 for LTS2019 right?

Reply:
Either 10.10 or LTS-2019 if pulled from Docker Hub with `docker pull nuxeo` (official image, but soon no more maintained), or 10.10-HFXX if pulled from the internal Docker registry, but not sure if want to advertise this hotfixed image used by Nuxeo Cloud :thinking: